content/2017/1/25/parallax.rst

```diff
@@ -0,0 +1,36 @@
+public: yes
+author: sondra
+tags: [design]
+image:
+  - src: '2017/parallax/parallax.jpg'
+summary: |
+  Whether you like it not, parallax scrolling is a reality, so let's learn how to do it well with CSS 3D. Paul Lewis' guide to performant parallaxing is the perfect place to start.
+
+
+Let's Talk About Parallax: Dos and Don'ts
```

Comment:
I wonder if we want a title pattern for external link posts. Something like: "Link: Parallax Dos and Don'ts". I keep feeling weird about just using their title directly.